install/kubernetes/helm/istio-remote/values.yaml

```diff
@@ -4,22 +4,22 @@
 
 global:
   # The Istio control plane Pilot endpoint
-  pilotEndpoint: istio-pilot.istio-system
+  pilotEndpoint:
```

Comment:
Is this a problem not defining a pilotEndpoint value?   I am not a helm expert but it looks fishy since this is only defined in istio-remote

Reply:
Ignore my comment got a bit of tutorial on how these values are used to build the charts